test/UnitTest/Microsoft.AspNet.OData.Test/PublicApi/Microsoft.AspNet.OData.PublicApi.bsl

```diff
@@ -2296,9 +2321,24 @@ public class Microsoft.AspNet.OData.Query.DefaultQuerySettings {
 	bool EnableFilter  { public get; public set; }
 	bool EnableOrderBy  { public get; public set; }
 	bool EnableSelect  { public get; public set; }
+	bool EnableSkipToken  { public get; public set; }
 	System.Nullable`1[[System.Int32]] MaxTop  { public get; public set; }
 }
 
+public class Microsoft.AspNet.OData.Query.DefaultSkipTokenHandler : SkipTokenHandler {
+	public DefaultSkipTokenHandler ()
+	public DefaultSkipTokenHandler (char delimiter)
```

Comment:
Not fond of having a public constructor just for test purposes. The test that uses this just replaces the delimiter; can we create a new, more interesting, test that doesn't rely on this constructor?